tripal_chado/src/TripalStorage/ChadoRecords.php

```diff
@@ -206,8 +213,8 @@ protected function initTable($elements) : bool {
         'link_columns' => [],
 
         // Aliases for columns. This is indexed by the column alias. The value
-        // is a set of key value pairs indicating the chado_table, table_alias and
-        // chado column names.
+        // is a set of key value pairs indicating the chado_table, table_alias
```

Comment:
```suggestion
        // is a set of key value pairs indicating the chado_table, table_alias,
```
Just to be annoying I insert an oxford comma here 😂